src/components/ProductContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import ProductCard from './ProductCard';
+import axios from 'axios'
+
+function ProductContainer() {
+    const [product,setProduct] = useState([])
+    useEffect(()=>{
+      (async function (){
+        try{
+        const {products} = await axios.get(
+          "/api/products").then(response => response.data);
```

Comment:
No need to use `then(response => response.data)` 
```
const {products} = await axios.get(
          "/api/products").data
```

Reply:
Cool Thanks